easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,313 @@
+name = 'Python'
+version = "3.7.4"
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.%(namelower)s.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['d63e63e14e6d29e17490abbe6f7d17afb3db182dbd801229f14e55f4157c4ba3']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+]
+dependencies = [
+    ('bzip2', '1.0.6'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.0'),
+    ('ncurses', '6.1'),
+    ('SQLite', '3.27.2'),
+    ('XZ', '5.2.4'),
+    ('GMP', '6.1.2'),  # required for pycrypto
+    ('libffi', '3.2.1'),  # required for cryptography
+]
```

Comment:
We should keep the comments for older OS'es that build OpenSSL.

Reply:
That was absent in 3.7.2 already. That's why it's missing

Reply:
No? https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/p/Python/Python-3.7.2-GCCcore-8.2.0.eb#L29

Reply:
interesting
in my local version it was missing actually
anyways... I'll add it again